flex-box-template-style.css

```diff
@@ -0,0 +1,117 @@
+.layout{
+    display:flex;
+    justify-content: center;
+    margin: 0 auto;
+    width: 100%;
+    height: 100%;
+    flex-direction: column;
+
+
+}
+.container{
+    display:flex;
+    width: 700px;
+    height:auto;
+}
+.nav{
+    display:flex;
+    flex-direction: row;
+    font-family: Helvetica Neue;
+}
+
+.right-nav{
+    display:flex;
+    width: 400px;
+    justify-content: right;
+    font-size: 20px;
+    align-items: center;
+}
+.right-nav-item{
+    flex:0 0 100px;
+    text-align: right;
+}
+
+.left-nav{
+    display:flex;
+    width: 300px;
+    font-size: 25px;
+    align-items: center;
+}
+.header a{
+    color: rgb(66,66,66);
+}
+a{
+    text-decoration: none;
+    color:cornflowerblue;
+}
+.content{
+    display:flex;
+    flex-direction: row;
+    padding-top: 20px;
+}
+.research-content{
+    display:flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    font-family: Arial, Helvetica, sans-serif;
+
+}
+.software-content{
+    display:flex;
+    flex-direction:column;
+}
+.content-paragraph{
+    display:flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    font-family: Arial, Helvetica, sans-serif;
+}
+p{
+    text-align: justify;
+    font-size: 16px;
+    margin: 3px 0;
+}
+.header{
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    height: 50px;
+    border-bottom: 1px solid rgb(217,217,217);
+    background-color: rgb(247,247,247);
+}
+.main{
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    height: 500px;
```

Comment:
If we are putting only one item in the container and there is no special reason, we do not give a fixed height. because we do not want it to break our structure on different screens.